test-e2e.sh

```diff
@@ -85,10 +85,28 @@ EXCLUDED_TARGETS="-//images/gcloud-bazel:gcloud_install -//images/gcloud-bazel:g
 bazel build -- //... $EXCLUDED_TARGETS
 bazel test  -- //... $EXCLUDED_TARGETS
 
+# Create a unique namespace for this job using the repo name and the build id
+export E2E_NAMESPACE="build-${BUILD_ID:-0}"
+
+kubectl get "namespaces/${E2E_NAMESPACE}" &> /dev/null || kubectl create namespace "${E2E_NAMESPACE}"
+
+delete() {
+    # Delete the namespace
+    echo "Deleting kubernetes namespace ${E2E_NAMESPACE}"
+    # Delete the namespace
+    kubectl delete namespaces/$E2E_NAMESPACE
```

Comment:
`kubectl delete namespaces/$E2E_NAMESPACE &>/dev/null` to silence non-fatal errors?